lib/protobuf/field/message_field.rb

```diff
@@ -9,7 +9,7 @@ class MessageField < BaseField
       #
 
       def acceptable?(val)
-        unless val.instance_of?(type_class) || val.respond_to?(:to_hash)
+        unless val.is_a?(type_class) || val.is_a?(Hash)
```

Comment:
The only reason I think that's ok is because you could do `OtherBasicMessage.new.to_hash` and pass it down... the message will ignore any fields it doesn't know about so it seems ok to me. Thoughts?